wdl/pipelines/ONT/MultiAnalysis/ONTPfHrp2Hrp3Status.wdl

```diff
@@ -86,7 +85,7 @@ task IsLocusDeleted {
         boot_disk_gb:       10,
         preemptible_tries:  2,
         max_retries:        1,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-mosdepth:0.3.1"
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-mosdepth:0.3.2"
```

Comment:
I know we have `0.3.2` on GCR, but given that the `main` branch is using `0.3.1`, we can do one of two things here:

- not updating the version here
- find out what `0.3.2` changed from `0.3.1`. I see you included several more packages in `lr-mosdepth`, so if you built that, then let's also bump the version in the makefile in that docker foler.

Reply:
Makefile version bumped.